src/libraries/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -8,6 +8,14 @@
 using System.Runtime.Intrinsics;
 using System.Runtime.Intrinsics.X86;
 using System.Runtime.Intrinsics.Arm;
+using System;
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Numerics;
+using System.Text;
```

Comment:
nit: We generally try to keep using's sorted alphabetically and grouped together.

Unnecessary usings should be removed.

Reply:
Removed